troubleshooting/known_issues/kubernetes_node_rootFS_out_of_space.md

```diff
@@ -0,0 +1,26 @@
+# Kubernetes Master or Worker node's root filesystem is out of space
+
+## Description
+
+ There is a known bug in kubernetes 1.19.9 where movement of a pod with an attached volume may not complete in time and cause the kubelet service to stream error messages to the /var/log/messages log file.  If this goes unchecked it will fill up the root file system.
```

Comment:
I would add a comma between "goes unchecked" and "it will"

Reply:
updated